sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -470,6 +493,14 @@ public final int appendBooleans(int count, boolean v) {
     return result;
   }
 
+  public final int appendBooleans(int count, int src, int offset) {
```

Comment:
nit: add some comments to this method, it is a bit confusing that the input is a int.

Reply:
Updated